subiquity/ui/views/ubuntu_pro.py

```diff
@@ -206,47 +461,207 @@ def show_unknown_error(self) -> None:
+
+class TokenAddedWidget(Stretchy):
+    """ Widget that shows that the supplied token is valid and was "added".
+    +---------------- Token added successfully ---------------+
+    |                                                         |
+    | Your token has been added successfully and your         |
+    | subscription configuration will be applied at the first |
+    | boot.                                                   |
+    |                                                         |
+    |                       [ Continue ]                      |
+    +---------------------------------------------------------+
+    """
+    def __init__(self, parent: UbuntuProView,
+                 on_continue: Callable[[], None]) -> None:
+        """ Initializes the widget. """
+        self.parent = parent
+        cont = done_btn(
+                label=_("Continue"),
+                on_press=lambda unused: on_continue())
+        widgets = [
+            Text(_("Your token has been added successfully and your"
+                   " subscription configuration will be applied at the first"
+                   " boot.")),
+            Text(""),
+            button_pile([cont]),
+            ]
+        super().__init__("Token added successfully", widgets,
+                         stretchy_index=0, focus_index=2)
+
+
+class AboutProWidget(Stretchy):
+    """ Widget showing some information about what Ubuntu Pro offers.
+    +------------------- About Ubuntu Pro --------------------+
+    |                                                         |
+    | Ubuntu Pro subscription gives you access to multiple    |
+    | security & compliance features such as:                 |
+    |                                                         |
+    | * Security patching for over 30.000 packages, with a    |
+    | focus on High and Critical CVEs (extended from 2.500)   |
+    | * ...                                                   |
+    | * ...                                                   |
+    |                                                         |
+    | It's free for personal use on up to 3 machines.         |
+    | More information on ubuntu.com/pro                      |
+    |                                                         |
+    |                       [ Continue ]                      |
+    +---------------------------------------------------------+
+    """
+    def __init__(self, parent: UbuntuProView) -> None:
+        """ Initializes the widget."""
+        self.parent = parent
+
+        ok = ok_btn(label=_("Continue"), on_press=lambda unused: self.close())
+
+        title = _("About Ubuntu Pro")
+        header = _("Ubuntu Pro subscription gives you access to multiple"
+                   " security & compliance features such as:")
+
+        services = [
+            _("Security patching for over 30.000 packages, with a focus on"
+              " High and Critical CVEs (extended from 2.500)"),
```

Comment:
The indent here between bullet points when we need a second line for a single bullet point doesn't quite match mockups.   I appreciate that this is probably non-trivial.  Thoughts?

Reply:
I found a solution to achieve that using a `Columns` construct. If the bullet point + text with indent is something we need for other screens going forward, I can move it somewhere under `subiquitycore.ui`.

Reply:
Screenshot updated.

Reply:
Nice!